### PR TITLE
nginx-ingress migration

### PR DESCRIPTION
**Pull request by SirTyson**
This PR adds support for `nginx-ingress` to supercluster. Now supercluster is compatible with both Traefik and Nginx ingresses. To use `nginx-ingress`, [the stellar fork](https://github.com/stellar/ingress-nginx) of Nginx must be deployed on the cluster. Currently, local k3s development does not support Nginx.

To enable `nginx-ingress`, use the flag `--ingress-class nginx-private`.

### CODE REVIEW

**Review by satyamz — APPROVED** [review]
Ingress configuration looks good to me. I'd defer to someone from core team for the code review.

**Review by sisuresh — APPROVED** [review]
LGTM

**Comment by SirTyson** [comment]
`nginx-ingress` should now work locally with `k3s`. You can follow the updated instructions in `k3s.md` to test.

File: src/FSLibrary/StellarKubeSpecs.fs
Comment (satyamz) [inline src/FSLibrary/StellarKubeSpecs.fs:840]: Since, we are planning to merge this change after we've migrated existing ingress class from `nginx-private` to `private`. We should keep this as `private`.

**Pull request merged**
